### PR TITLE
Gui 5.0.2 UI Improvements

### LINKED ISSUE #899: GUI Objects Flickering after Screen Resized
## Problem

The objects "flicker" when resized, but stop once the mouse is released.

Clicking the mouse clearly solves the problem, and the logic should be applied somewhere else.

## Expected 

Expected: Resize GUI without UI objects flickering

## Operating System and Version

All OS

## GUI Version

5.0.1

## Running standalone app

Standalone and development branch

## Type of OpenBCI Board

All data modes once session is started

## Are you using a WiFi Shield?

No

## Console Log

The following code should be removed in Interactivity.pde, since the mouse is released outside of the app when resizing the app.
```
    if (settings.screenHasBeenResized) {
        println("OpenBCI_GUI: mouseReleased: screen has been resized...");
        settings.screenHasBeenResized = false;
    }
```

### PR DESCRIPTION
**Pull request by retiutut**
- [x] Default to Body uV Gain Mode
- [x] Optimize TimeSeries Y Axis Autoscale
- [x] Add IP address for NovaXR in control panel
- [x] Update ChannelSelect feature for all widgets
- [x] Show if channel is active in the channelSelect using toggle border colors
- [x] Fix #899 
- [x] Put HardwareSettings Labels at the top, and command buttons at the bottom

### CODE REVIEW

File: release_script/index.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
    //var AUTO_TITLE = true;
    //var S3_REGION = 's3'; // for us-east-1
 </script>
-<script type="text/javascript" src="https://rufuspollock.github.io/s3-bucket-listing/list.js"></script>
```
Comment (retiutut) [inline release_script/index.html:21]: A stable, working copy of this Javascript is now stored in S3 root directory.

File: OpenBCI_GUI/BoardCyton.pde [hunk 2]
```diff
@@ -269,7 +269,7 @@ implements ImpedanceSettingsBoard, AccelerometerCapableBoard, AnalogCapableBoard
 
         // The command 'd' is automatically sent by brainflow on prepare_session
         currentADS1299Settings = new CytonDefaultSettings(this);
-        useDynamicScaler = false;
+        setUseDynamicScaler(true);
```
Comment (Andrey1994) [inline OpenBCI_GUI/BoardCyton.pde:272]: maybe I forgot smth about Java but as I remember its not good to call methods from constructor. lets replace it to manual variable initialization

File: OpenBCI_GUI/BoardNovaXR.pde [hunk 3]
```diff
@@ -152,22 +152,23 @@ implements ImpedanceSettingsBoard, EDACapableBoard, PPGCapableBoard, BatteryInfo
...
 
+        ipAddress = _ip;
         initialSettingsMode = mode;
         sampleRate = _sampleRate;
         samplingRateCache = sampleRate.getValue();
 
         // store a copy of the default settings. This will be used to undo brainflow's
         // gain scaling to re-scale in gui
         defaultSettings = new NovaXRDefaultSettings(this, NovaXRMode.DEFAULT);
+        setUseDynamicScaler(true);
```
Comment (Andrey1994) [inline OpenBCI_GUI/BoardNovaXR.pde:171]: the same here

File: OpenBCI_GUI/BoardNovaXR.pde [hunk 4]
```diff
@@ -2,8 +2,7 @@ import brainflow.*;
 
 import org.apache.commons.lang3.ArrayUtils;
 
-final boolean novaXREnabled = false;
-final boolean usingNovaXREmulator = false;
+final boolean novaXREnabled = true;
```
Comment (Andrey1994) [inline OpenBCI_GUI/BoardNovaXR.pde:5]: dont forget to set it to false

File: OpenBCI_GUI/SessionSettings.pde [hunk 5]
```diff
@@ -495,15 +502,15 @@ class SessionSettings {
         JSONArray saveActiveChanSpectTop = new JSONArray();
         int numActiveSpectChanTop = w_spectrogram.spectChanSelectTop.activeChan.size();
         for (int i = 0; i < numActiveSpectChanTop; i++) {
-            int activeChan = w_spectrogram.spectChanSelectTop.activeChan.get(i) + 1; //add 1 here so channel numbers are correct
+            int activeChan = w_spectrogram.spectChanSelectTop.activeChan.get(i);
```
Comment (Andrey1994) [inline OpenBCI_GUI/SessionSettings.pde:505]: I dont understand why its wo +1 now. I suppose its correct but dont get why its changed

**Review by Andrey1994 — APPROVED** [review]

File: OpenBCI_GUI/BoardCyton.pde (hunk 2 above)
Comment (retiutut) [reply OpenBCI_GUI/BoardCyton.pde:272]: https://stackoverflow.com/questions/18348797/why-is-it-considered-bad-practice-in-java-to-call-a-method-from-within-a-constru

File: OpenBCI_GUI/SessionSettings.pde (hunk 5 above)
Comment (retiutut) [reply OpenBCI_GUI/SessionSettings.pde:505]: I will look at this again...

**Pull request merged**